### PR TITLE
chore: bump marked version

### LINKED ISSUE #1726: posibility of a 2.0.0 release with updated marked
Hi, @RomanHotsiy,
### Issue Description

   I noticed that a vulnerability is introduced in **_redoc@2.0.0-rc.45_**:
     Vulnerability **SNYK-JS-MARKED-584281** affects package **_marked_** (versions:<1.1.1):  [https://snyk.io/vuln/SNYK-JS-MARKED-584281](https://snyk.io/vuln/SNYK-JS-MARKED-584281)
  The above vulnerable package is referenced by **_redoc@2.0.0-rc.45_** via:
`redoc@2.0.0-rc.45 ➔ marked@0.7.0`

Since **_redoc@2.0.0-rc.45_** ([13,275 downloads per week](https://www.npmjs.com/package/redoc/v/2.0.0-rc.45?activeTab=versions)) is referenced by **101** downstream projects (e.g.,redoc-cli 0.12.2, @adobe/parliament-ui-components 4.6.2, docusaurus-theme-redoc 0.4.4, redocusaurus 0.4.4, gh-openapi-docs-ts 0.3.6), the vulnerability [**SNYK-JS-MARKED-584281**](https://snyk.io/vuln/SNYK-JS-MARKED-584281) can be propagated into these downstream projects and expose security threats to them via the following package dependency paths:
 (1)`@ga4gh/gh-openapi-docs@0.2.2-rc3 ➔ redoc-cli@0.9.13 ➔ redoc@2.0.0-rc.45 ➔ marked@0.7.0`
(2)`@abios/abios-redoc-cli@0.13.2 ➔ redoc@2.0.0-rc.45 ➔ marked@0.7.0`
  **......**

If **_redoc@2.0.0-rc.45_** removes the vulnerable package from the above version, then its fixed version can help downstream users decrease their pain.

**Given the large number of downstream users, could you help update your package to remove the vulnerability from redoc@2.0.0-rc.45 ?**

### Fixing suggestions

In **_redoc@2.0.0_**, maybe you can kindly try to perform the following upgrade :
`marked ^0.7.0 ➔ ^1.1.1`; 

**Note:**
_**marked@1.1.1**(>=1.1.1) has fixed the vulnerability [**SNYK-JS-MARKED-584281**](https://snyk.io/vuln/SNYK-JS-MARKED-584281)._

Thank you for your attention to this issue and welcome to share other ways to resolve the issue.^_^

### PR DESCRIPTION
**Pull request by Oprysk**
## What/Why/How?
fixes #1726

bump `marked` version
## Check yourself

- [x] Code is linted
- [x] Tested
- [ ] All new/updated code is covered with tests

### CODE REVIEW

File: src/services/MarkdownRenderer.ts [hunk 1]
```diff
@@ -58,9 +59,11 @@ export class MarkdownRenderer {
 
   private headingEnhanceRenderer: marked.Renderer;
   private originalHeadingRule: typeof marked.Renderer.prototype.heading;
+  parser: marked.Parser;
 
   constructor(public options?: RedocNormalizedOptions) {
     this.headingEnhanceRenderer = new marked.Renderer();
+    this.parser = new marked.Parser();
```
Comment (RomanHotsiy) [inline src/services/MarkdownRenderer.ts:66]: It seems it is not used anywhere... ?

File: src/components/Responses/styled.elements.ts [hunk 2]
```diff
@@ -11,14 +11,12 @@ export const StyledResponseTitle = styled(ResponseTitle)`
...
   margin-bottom: 4px;
   line-height: 1.5em;
-  background-color: #f2f2f2;
   cursor: pointer;
 
   color: ${props => props.theme.colors.responses[props.type].color};
   background-color: ${props => props.theme.colors.responses[props.type].backgroundColor};
   &:focus {
-    outline: auto;
-    outline-color: ${props => props.theme.colors.responses[props.type].color};
```
Comment (RomanHotsiy) [inline src/components/Responses/styled.elements.ts:21]: can we move those changes into separate PR maybe?

**Review by RomanHotsiy — CHANGES_REQUESTED** [review]

File: src/services/MarkdownRenderer.ts (hunk 1 above)
Comment (Oprysk) [reply src/services/MarkdownRenderer.ts:66]: it is used by `marked.Renderer()` under the hood. Without this initialization, `marked` will not have context of the parser

File: src/components/Responses/styled.elements.ts (hunk 2 above)
Comment (Oprysk) [reply src/components/Responses/styled.elements.ts:21]: ok, will revert that

File: src/services/MarkdownRenderer.ts (hunk 1 above)
Comment (RomanHotsiy) [reply src/services/MarkdownRenderer.ts:66]: Add a comment please. Otherwise someone will remove it 😄

**Review by RomanHotsiy — APPROVED** [review]
